tests/core/manifest/ThemeResourceLoaderTest.php

```diff
@@ -75,6 +75,54 @@ public function testFindNestedThemeTemplates() {
 		);
 	}
 
+	public function findTemplateByType() {
```

Comment:
No wonder this test wasn't running... the method wasn't named `test*`... will fix.